src/appleseed.shared/application/progresstilecallback.cpp

```diff
@@ -134,20 +135,28 @@ namespace
 // ProgressTileCallbackFactory class implementation.
 //
 
+struct ProgressTileCallbackFactory::Impl
+{
+    std::unique_ptr<renderer::ITileCallback> m_callback;
```

Comment:
`std::` qualifiers are redundant in this file since there's a `using namespace std` clause at the top.

Reply:
And so are `renderer::` qualifiers.

Reply:
Oops, my bad! I didn't see those statements there.